content/docs/2.5/operate/querying.md

```diff
@@ -0,0 +1,65 @@
++++
+title = "Querying KEDA metrics"
+description = "Guidance for querying KEDA metrics"
+weight = 100
++++
+
+## Querying KEDA metrics
+
+The metrics exposed by KEDA Metrics Adapter can be queried directly using `kubectl`:
+```bash
+kubectl get --raw "/apis/external.metrics.k8s.io/v1beta1"
+```
+
+This will return a json with the list of metrics exposed by KEDA:
+```json
+{
+  "kind": "APIResourceList",
+  "apiVersion": "v1",
+  "groupVersion": "external.metrics.k8s.io/v1beta1",
+  "resources": [
+    {
+      "name": "s0-rabbitmq--mymetric",
+      "singularName": "",
+      "namespaced": true,
+      "kind": "ExternalMetricValueList",
+      "verbs": [
+        "get"
+      ]
+    },
+    {
+      "name": "s1-rabbitmq--mymetric2",
+      ....
+    }
+  ]
+}
+```
```

Comment:
Might be nice to add a note that metric name for a specific ScaledObject can be get from from ScaledObject's Status with some example.